create-convex/README.md

```diff
@@ -13,7 +13,7 @@ via additional command line options. For example, to scaffold a Convex project
 with no client, run:
 
 ```bash
-npm create convex@latest my-app -- -t bare
+npm create convex@latest my-app -t bare
```

Comment:
This doesn't work. You need `--` if you're using `npm create`. Did you perhaps test with `npx create-convex` or via node?

I suspect that this confusion is why Next.js uses the `npx create-next` variant instead of `npm create next`. But Vite uses this variant so that's I'm modeling after - plus specifying the template is advanced usecase.

Reply:
No I was definitely using `npm create` pulling these commands verbatim from the readme, and seeing the behavior described (template flag ignored), which also matches what the minimist docs describe as the behavior for `--`. It sounds like you're seeing different behavior though?  🤔

Reply:
Hm OK I think I know what happened, after I had fixed the other problem with 0.0.5 and was testing my edited version of the script with Node must have been when I saw this behavior. Seems to be working fine with 0.0.6, so nevermind!